core/ts/Poll.ts

```diff
@@ -399,7 +404,7 @@ export class Poll implements IPoll {
 
     // Only allow one poll to be processed at a time
     if (this.maciStateRef.pollBeingProcessed) {
-      assert(this.maciStateRef.currentPollBeingProcessed === pollId);
+      assert(this.maciStateRef.currentPollBeingProcessed === pollId, "Another poll is currently being processed");
     }
 
     if (this.numBatchesProcessed === 0) {
```

Comment:
Thinking about if `r===0` --> `this.messages.length` is multiple of `batchSize` --> then doing `Math.floor(this.messages.length / batchSize) * batchSize` actually equal to `this.messages.length`?

Reply:
that's a great catch, think we can just remove that altogether